src/types/types.ts

```diff
@@ -5,6 +5,16 @@ export interface UploadOptions  {
   onPreview: (src: string) => void 
 };
 
+/**
+ * Represents the format of a file object data with the additional data.
+ */
+export type FileObjectData<AdditionalData = {}> = {
```

Comment:
why this type is not reused below?

Reply:
it was initially defined to be used to defined parameters types of member functions. But it can be used in the interfaces if you would like to.